src/main/java/com/orasi/bluesource/Accounts.java

```diff
@@ -376,5 +396,137 @@ public void selectCell(int row, int column) {
 		PageLoaded.isDomComplete(driver, 1);
 	}
 	
+	/**
+	 * Clicks on New Project, fills out required fields and submits form
+	 * 
+	 * @author Christopher Batts
+	 */
+	public void addProject() {
+		btnNewProject.click();
+		txtProjectName.sendKeys("New Project 1");
+		txtProjectStart.sendKeys("04062018");
+		txtProjectEnd.sendKeys("06042018");
+		btnCreateProject.click();
+	}
+	
+	/**
+	 * Clicks a project link in the projects table under an account.
+	 * 
+	 * @param String Name of project to select
+	 * @author Christopher Batts
+	 */
+	public void selectProject(String projName) {
+		PageLoaded.isDomComplete(driver, 5);
+		tblProjects.findElement(By.linkText(projName)).syncVisible(5);
+		tblProjects.findElement(By.linkText(projName)).click();
+	}
+	
+	/**
+	 * Clicks the New Role button under a project
+	 * 
+	 * @author Christopher Batts
+	 */
+	public void clickNewRole() {
+		btnNewRole.syncVisible(5);
+		btnNewRole.click();
+	}
+	
+	/**
+	 * Clicks the Billable/Non-Billable toggle
+	 * 
+	 * @author Christopher Batts
+	 */
+	public void toggleBill() {
+		btnBill.syncVisible(5);
+		btnBill.click();
+	}
+	
+	/**
+	 * Checks state of Billable/Non-Billable toggle. Returns true if Billable is displayed, otherwise returns false.
+	 * 
+	 * @return Boolean 
+	 * @author Christopher Batts
+	 */
+	public boolean isBillOn() {
+		return !(btnBill.getAttribute("class").contains("off"));
+	}
+	
+	/**
+	 * Clicks the +New Role button, fills out required fields, and submits the form.
+	 * 
+	 * @author Christopher Batts
+	 */
+	public void createRole() {
```

Comment:
similar comment about data driving here.  the method parameters should be something like roleType and maxResources, so that your tests can change the type of role created if needed.